src/cmdlib.sh

```diff
@@ -255,6 +255,34 @@ EOF
     return "$(cat "${workdir}"/tmp/rc)"
 }
 
+openshift_git_hack() {
+    # When OPENSHIFT_GIT_HACK is defined as a build environment variable,
+    # this will fetch the missing .git into the directory where its expected.
+
+    # OpenShift 3.9 does not expose the GIT Repo. This populates a .git in ${gitd}.
+    # See https://github.com/coreos/coreos-assembler/issues/320
+    # TODO: Remove this when https://github.com/openshift/source-to-image/issues/457
+    #       is resolved.
+
+    local gitd=${1}; shift;
+
+    if [ "${OPENSHIFT_GIT_HACK:-x}" == "x" ]; then
+        return
+    fi
+
+    info "NOTICE: using workaround for Openshift missing .git"
+
+    if [ -d "${gitd}/.git" ]; then
+        info "Hack is not needed, .git exists"
```

Comment:
Let's move this above the `info` above, and don't print anything if there isn't an issue?

Reply:
Just pushed a fix.